.github/workflows/e2e-tests.yml

```
@@ -0,0 +1,221 @@
+name: E2E Tests
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)